### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by Fan-iX** [shenwei356/seqkit#424]
### Prerequisites

- [x] make sure you're are using the latest version by `seqkit version`
  - `seqkit version`: `seqkit v2.6.0`
- [x] read the [usage](http://bioinf.shenwei.me/seqkit/usage/)

### Describe your issue

All of the following commands works well:
```
seqkit seq <(echo -e "") <(echo -e ">seq2\nCGATA")
seqkit seq <(echo -e ">seq2\nCGATA") <(echo -e "" | seqkit seq)
cat <(echo -e "" | seqkit seq) <(echo -e ">seq2\nCGATA")
```

But
```
seqkit seq <(echo -e "" | seqkit seq) <(echo -e ">seq2\nCGATA")
```
raises error
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x76a3a2]

goroutine 1 [running]:
github.com/shenwei356/bio/seqio/fastx.NewReader(0x0, {0x7ffea84f1f5d, 0xa}, {0xb2a6ff?, 0x9?})
        /home/shenwei/go/pkg/mod/github.com/shenwei356/bio@v0.9.1/seqio/fastx/reader.go:134 +0x362
github.com/shenwei356/seqkit/v2/seqkit/cmd.glob..func28(0xc0002e8000?, {0xc000113320, 0x2, 0x2})
        /home/shenwei/go/src/github.com/shenwei356/seqkit/seqkit/cmd/seq.go:240 +0x1326
github.com/spf13/cobra.(*Command).execute(0x155dba0, {0xc0001132c0, 0x2, 0x2})
        /home/shenwei/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0x155a220)
        /home/shenwei/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        /home/shenwei/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/shenwei356/seqkit/v2/seqkit/cmd.Execute()
        /home/shenwei/go/src/github.com/shenwei356/seqkit/seqkit/cmd/root.go:71 +0x1a
main.main()
        /home/shenwei/go/src/github.com/shenwei356/seqkit/seqkit/main.go:58 +0xf
```

<details>
<summary>As for why I found this ...</summary>

I want to selectively reverse-complement some sequences (without creating too much intermediate files), so I ran
```
seqkit seq <(
  seqkit grep -f <(awk '$2=="-"{print $1}' orientation.txt) sequences.fa | seqkit seq -rp -t DNA
) <(
  seqkit grep -f <(awk '$2!="-"{print $1}' orientation.txt) sequences.fa
)
```
and when all lines in `orientation.txt` gives a `+` in the second column, I got this error.

</details>

**Comment by shenwei356** [comment]
Thanks for reporting. Actually, I've found it last week. Here's the new version, I just released: https://github.com/shenwei356/seqkit/releases/tag/v2.6.1

>  fix panic of nil pointer introduced in v2.6.0, which happens when handling multiple input files and some of them have file sizes of zero.

Thank you again.

**Comment by Fan-iX** [comment]
The new version works well.
Thank you.
